schedule/yast/autoupgrade_textmode.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: schedule/yast/autoupgrade_textmode.yaml
+description: >
+  Performs a migration with textmode.
+schedule:
+  - yam/migration/setup_upgrade_env
+  - autoyast/prepare_profile
+  - installation/bootloader_start
+  - autoyast/installation
+  - installation/first_boot
+  - console/system_prepare
+  - yam/migration/restore_upgrade_env
```

Comment:
can we run this one up, after first_boot, would make more sense.
If not, then put it the last.

Reply:
updated, I re-trigger the test on x86_64, https://openqa.suse.de/tests/18097202# although I think won't affect the result.